ContextDesktop/src/main/java/com/github/migi_1/Context/vr/VRHandler.java

```diff
@@ -0,0 +1,92 @@
+package com.github.migi_1.Context.vr;
+
+
+
+import com.github.migi_1.Context.model.World;
+import com.jme3.input.InputManager;
+import com.jme3.input.KeyInput;
+import com.jme3.input.controls.ActionListener;
+import com.jme3.input.controls.KeyTrigger;
+import com.jme3.renderer.RenderManager;
+
+import jmevr.app.VRApplication;
+
+/**
+ * VR handler for the current world.
+ * @author NilsHullegien
+ */
+public class VRHandler extends VRApplication {
+
+    // set some VR settings & start the app
+    public static void main(String[] args){
+        VRHandler vrh = new VRHandler();
+        //test.preconfigureVRApp(PRECONFIG_PARAMETER.USE_STEAMVR_COMPOSITOR, false); // disable the SteamVR compositor (kinda needed at
+        vrh.preconfigureVRApp(PRECONFIG_PARAMETER.FLIP_EYES, false);
+        vrh.preconfigureVRApp(PRECONFIG_PARAMETER.SET_GUI_OVERDRAW, true); // show gui even if it is behind things
+        vrh.preconfigureVRApp(PRECONFIG_PARAMETER.INSTANCE_VR_RENDERING, false); // faster VR rendering, requires some vertex shader changes (see jmevr/shaders/Unshaded.j3md)
+        vrh.preconfigureVRApp(PRECONFIG_PARAMETER.NO_GUI, false);
+        vrh.preconfigureFrustrumNearFar(0.1f, 512f); // set frustum distances here before app starts
+        //test.preconfigureResolutionMultiplier(0.666f); the moment)
+        vrh.preconfigureVRApp(PRECONFIG_PARAMETER.USE_CUSTOM_DISTORTION, false); // use full screen distortion, maximum FOV, possibly quicker (not compatible with instancing)
+        vrh.preconfigureVRApp(PRECONFIG_PARAMETER.ENABLE_MIRROR_WINDOW, false); // runs faster when set to false, but will allow mirroring
+        vrh.preconfigureVRApp(PRECONFIG_PARAMETER.FORCE_VR_MODE, false); // render two eyes, regardless of SteamVR
+        vrh.preconfigureVRApp(PRECONFIG_PARAMETER.SET_GUI_CURVED_SURFACE, true);// you can downsample for performance reasons
+        vrh.start();
+    }
+
```

Comment:
I recommend splitting up the main method, putting all the configurations into a separate function looks a lot cleaner